packages/map-template/src/components/Map/MapboxMap/MapboxMap.jsx

```diff
@@ -21,6 +23,7 @@ function MapboxMap({ onMapView, onPositionControl }) {
     const [hasPositionControl, setHasPositionControl] = useState(false);
     const mapsIndoorsInstance = useRecoilValue(mapsIndoorsInstanceState);
     const primaryColor = useRecoilValue(primaryColorState);
+    const [, setMapboxSessionToken] = useRecoilState(mapboxSessionTokenState);
```

Comment:
Why was it that we didn't want to store this in the browser's sessionStorage (https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)? Then the session will survive accross reloads, which may be what we want.

Reply:
Hmm, I remember we wanted to store the value in an atom because it made more sense to pass it down to the component. 🤔

Reply:
It would be equally easy to read values from the sessionStorage. Atoms are like "lil session" 😝

Reply:
Lil session 😅

Reply:
Do you think we should change it then?

Reply:
Yes, I think using sessionStorage to store a session token is more correct.

Reply:
Okay... good bye lil session

Reply:
The grown up session is taking over